### PR TITLE
Add config_yaml samples

### PR DESCRIPTION
**Pull request by YarBor**
Add config_yaml samples , use new api to load config.yaml  #697

### CODE REVIEW

**Review by FinalT — COMMENTED** [review]
Thanks! A few comments.

**Review by FinalT — APPROVED** [review]
LGTM.

File: config_yaml/README.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+		panic(err)
+	}
+	logger.Info("ConfigTest successfully")
+}
+
+```
+
+### 3.3 Show
+
+Start the server first and then the client. You can observe that the client prints `ConfigTest successfully` and the configuration is loaded and the call is successful.
```
[the remark this answers is not in the document]
Comment (YarBor) [reply config_yaml/README.md]: Thanks for reminding

**Comment by YarBor** [comment]
> @YarBor 处理完给我留个消息，我再 review 下
I think it is done

**Review by chickenlj — APPROVED** [review]
LGTM.

**Pull request merged**